.gitignore

```diff
@@ -19,11 +19,9 @@
 .DS_Store
 
 # CMake artifacts
-build/
-build-*/
+/build/
+/build-*/
 Testing/
-# Include iree.build package
-!compiler/bindings/python/iree/compiler/build/
```

Comment:
This is more nuanced, see https://github.com/iree-org/iree/pull/18630#discussion_r1781325856